spk/homeassistant/src/requirements.txt

```diff
@@ -0,0 +1,43 @@
+# required for Home Assistant core
+requests>=2.0
+#pyyaml>=3.11 is currently included as a module
+
+# optional, needed for specific components
+
+# discovery
+zeroconf>=0.16.0
+#cross-compiled dependency for zeroconf
+netifaces>=0.10.4
+
+# sun
+pyephem>=3.7
+
+# lights.hue
+phue>=0.8
+
+# media_player.cast
+# pychromecast>=0.5 disabled since it currently does not work with Chromecast api
+
+# keyboard
+# pyuserinput>=0.1.9 is excluded for the moment
+
+# switch.tellstick, sensor.tellstick
+# tellcore-py>=1.0.4 is excluded for the moment
+
+# device_tracker.nmap
+python-libnmap>=0.6.2
+
+# notify.pushbullet
+pushbullet.py>=0.7.1
+
+# thermostat.nest
+python-nest>=2.1
+
+# z-wave
+# pydispatcher>=2.0.5 is excluded for the moment
+
+# sensor.systemmonitor 
+# psutil>=2.2.1 is excluded for the moment
```

Comment:
Fyi, psutil also contains C-code. If you want to include it in the package, cross-compile it.